Create Symmetric Matrix.cpp

guptaa98 opened the pull request.

guptaa98 committed:
Create Symmetric Matrix.cpp

ighosh98 commented:
Add the problem link in the description and also on the file. Also, make the code readable

guptaa98 committed:
Update Symmetric Matrix.cpp

Please find the link of Problem below.
https://codeforces.com/problemset/problem/1426/B

guptaa98 commented:
Updated the code with the link in description. Please check it.

ighosh98 commented:
Looks fine
